src/javascript/app/pages/cashier/payment_agent_withdraw.js

```diff
@@ -52,14 +53,15 @@ const PaymentAgentWithdraw = (() => {
             }
             setActiveView(view_ids.form);
 
-            const currency = Client.get('currency');
-            const form_id  = `#${$(view_ids.form).find('form').attr('id')}`;
-            const min      = isCryptocurrency(currency) ? 0.002 : 10;
-            const max      = isCryptocurrency(currency) ? 5 : 2000;
+            const form_id           = `#${$(view_ids.form).find('form').attr('id')}`;
+            const is_crypto_not_dai = isCryptocurrency(currency) && currency !== 'DAI';
+            const min_withdrawal    = is_crypto_not_dai ? 0.002 : 10;
```

Comment:
what if we add a min_pa_withdrawl and max_pa_withdrawal to crypto_config in currency_base.js and add a function to get min pa withdrawal and max pa withdrawal? at least all the config will be in one place, and we won't need exceptions for DAI